csrankings-v.csv

```diff
@@ -221,7 +227,9 @@ Victor de Boer,VU Amsterdam,http://www.victordeboer.com,9Ck790YAAAAJ
 Victoria Interrante,University of Minnesota,http://www-users.cs.umn.edu/~interran,vQQOZVgAAAAJ
 Victoria Stodden,Univ. of Illinois at Urbana-Champaign,https://ischool.illinois.edu/people/victoria-stodden,LWw60SgAAAAJ
 Vida Dujmovic,University of Ottawa,http://cglab.ca/~vida,NOSCHOLARPAGE
+Vidhya V.,MAHE Manipal,https://www.manipal.edu/mit/department-faculty/faculty-list/vidhya-v/_jcr_content.html,pu9wkNcAAAAJ

```

Comment:
It does not appear to be a disambiguation entry but is identified as one.

https://dblp.org/pid/90/4421.html